sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -172,19 +172,23 @@ case class DescribeDatabaseCommand(
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val dbMetadata: CatalogDatabase =
       sparkSession.sessionState.catalog.getDatabaseMetadata(databaseName)
+    val allDbProperties = dbMetadata.properties
     val result =
       Row("Database Name", dbMetadata.name) ::
         Row("Description", dbMetadata.description) ::
-        Row("Location", CatalogUtils.URIToString(dbMetadata.locationUri)) :: Nil
+        Row("Location", CatalogUtils.URIToString(dbMetadata.locationUri))::
+        Row("Owner", allDbProperties.getOrElse("ownerName", "")) ::
+        Row("Owner Type", allDbProperties.getOrElse("ownerType", "")) :: Nil
```

Comment:
What if we simply show them as normal properties?

Reply:
If so，it is not only inconsist with hive and also how we show the owner of a table